bftengine/tests/testSeqNumForClientRequest/seqNumForClientRequest_test.cpp

```diff
@@ -1,8 +1,80 @@
 #include "gtest/gtest.h"
+#include "SimpleClientImp.cpp"
 
-TEST(seqNumForClientRequest_test, initial_test) {}
+using namespace std;
+using namespace bftEngine;
 
-int main(int argc, char* argv[]) {
+namespace {
+
+class mockSeqNumGenerator : public SeqNumberGeneratorForClientRequestsImp {
+ public:
+  uint32_t getLastCountOfUniqueFetchID() { return lastCountOfUniqueFetchID_; }
+
+  void setLastCountOfUniqueFetchID(uint64_t count) { lastCountOfUniqueFetchID_ = count; }
+};
+
+TEST(seqNumForClientRequest_test, send_request_fast_still_persistence) {
```

Comment:
what does "..fast_still_presistence" mean?

Reply:
The generated key is derieved from system clock, I have tested that the class generate unique SN, even if requests sent with no break between each other. 
I will think of some more meaningful name.